public/style/base/_typography.scss

```diff
@@ -56,3 +56,7 @@ a:visited {
   // box-shadow: 0px 1px 1px red;
   color: $brand-primary-darker;
 }
+
+a.blank:after {
+  content: '\01F5D7';
```

Comment:
Ha, how unfortunate that there isn't a great character for this. Let's go with `\0029C9` for the time being.